assets/blocks/course-list-block/index.js

```diff
@@ -161,10 +161,12 @@ const hideCourseListPatternsCarouselViewControl = () => {
 	const controls = document.querySelectorAll( `${ patternsControlClass }` );
 	controls.forEach( ( control ) => {
 		const controlButtons = control.querySelectorAll( 'button' );
-		// Hide carousel pattern view button.
-		controlButtons[ 0 ].style.display = 'none';
-		// Select Grid view button.
+		// Select Grid view.
 		controlButtons[ 1 ].click();
+
+		// Hide all control buttons.
+		controlButtons[ 0 ].style.display = 'none';
```

Comment:
Should we loop through them in case they decide to add more?

Reply:
If they decide to add more, we might have more problems than that :D since we can't distinguish between the grid and the carousel buttons (they look the same to us, except the icon and the text (and text is translatable)) so we actually use [1].click () to go to the grid view :D 

But yeah there isn't harm in hiding them in loop. I just wanted to make clear we only handle 2 buttons here.

Reply:
Here is a change: ee3af0cb311c4b2c3075d40979e6735684b1f5b3